tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -463,14 +463,14 @@
 
 	<!-- Executes accompanying unit tests -->
 	<target name="test" unless="test.skip">
-    <property runtime="runtime"/>
+		<property runtime="runtime"/>
 		<!-- run junit tests -->
 		<mkdir dir="${test.reports}" />
 		<!-- forkmode used to be "perBatch" on Java 1.8 using the util.IsolatedTestCaseRunner.
 		     This concept broken with Java11 for unknown reasons which is why forkmode has been
 		     changed to perTest to run each test in a separate VM.  This is slower compared to
 		     running all tests in a single VM. -->
-    <junit printsummary="yes" haltonfailure="${test.halt}" showoutput="no" haltonerror="${test.halt}"  forkmode="perTest" fork="yes" threads="${runtime.availableProcessors}">
+		<junit printsummary="yes" haltonfailure="${test.halt}" showoutput="no" haltonerror="${test.halt}"  forkmode="perTest" fork="yes" threads="${runtime.availableProcessors}">
```

Comment:
I'd prefer spaces over tabs, because some editors automagically replace tabs with `N` spaces, and `N` is not universially agree on.

Reply:
I agree although in this case using tabs matches the rest of the custombuild file